packages/block-library/src/page-list/edit.js

```diff
@@ -113,36 +113,16 @@ function BlockContent( {
 	}
 }
 
-function ConvertToLinks( { onClick, disabled } ) {
```

Comment:
This was in a different component to ensure that the modal didn't get rerendered whenever the page list renders.

Reply:
Yes but I can't call it from another block otherwise 🤷🏻

Reply:
From React's perspective, it would probably be better if the modal could be opened by the child instead, but I can see how it get's a bit muddy since we're dealing with blocks and portals.